dask/dataframe/tests/test_categorical.py

```diff
@@ -150,6 +150,7 @@ def test_concat_unions_categoricals():
     ],
 )
 @pytest.mark.filterwarnings("ignore:The default value of numeric_only")
+@pytest.mark.filterwarnings("ignore:The default of observed")
```

Comment:
Same question here; also, I notice that removing this `filterwarnings` we get a failure due to a groupby call in `value_counts_combine`; not sure if that's a place where we would want the warning to bubble up to users

Reply:
Good catch, this is not where we want it. I'll fix this test.